zulipterminal/ui_tools/buttons.py

```diff
@@ -109,14 +110,7 @@ def __init__(self, properties: List[Any],
         caption, self.stream_id, orig_color, is_private = properties
 
         # Simplify the color from the original version & add to palette
-        # TODO Should this occur elsewhere and more intelligently?
-        if len(orig_color) == 7:  # modern zulip server format
-            color = ''.join(orig_color[i] for i in (0, 1, 3, 5))
-        elif len(orig_color) == 4:  # possible with zulip servers <=1.9.0 ?
```

Comment:
We should probably retain these comments, at least as a pointer as to why this is necessary, or which versions may be relevant.

Reply:
Agreed. I'll move them to `canonicalized_color`